.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
+omit =
+    unstructured/ingest/*
```

Comment:
Why don't we want coverage on the ingest files?

Reply:
ingest is not the primary point of the repo yet, it's provided more as a way for users to quickly roll their own. and it is hard to test multiprocessing well with codecov (i think).

there is an an end-to-end test which IMO is more valuable than a bunch of unittests IMO. can prioritize unittests later?

Reply:
Yeah that works for me. Just with the cautionary note that I always find it slower to add unit tests after the fact